src/platform/windows/events_loop.rs

```diff
@@ -161,7 +347,8 @@ impl EventsLoop {
                 *context_stash.borrow_mut() = Some(ThreadLocalData {
                     sender: tx,
                     windows: HashMap::with_capacity(4),
-                    mouse_buttons_down: 0
+                    mouse_buttons_down: 0,
+                    file_drop_handlers: HashMap::new(),
```

Comment:
```suggestion
                    file_drop_handlers: HashMap::with_capacity(4),
```

It would be nice to build this with the same initial capacity as `windows`, assuming you're in agreement that it's a reasonable estimate to cover the majority of users.